README.md

```diff
@@ -128,3 +128,40 @@ Below is an example command
 ```sh
 docker run -v $PWD:$PWD -v /usr/local/bin:/usr/local/bin -w $PWD ghcr.io/gem5/gcn-gpu:v24-0 gem5-vega-se gem5/configs/example/apu_se.py -n 3 -c square
 ```
+
+## Note on running GPU FS
+
+You need not use docker to run simulations in GPU FS mode. You still need docker to compile applications like square. We will be simulating AMD's latest GPU, MI300, in FS mode
+Below is an example command
+
+```sh
+# To compile square
```

Comment:
I would put this in materials/isca24/10-gpufs/README.md, not in the base readme file.

Or, you could just have it in your slides.

Reply:
Will move it